content/docs/getting-started/connect-dbaas.md

```diff
@@ -64,3 +65,23 @@ Configure `cURL` with DBaaS credentials.
 ```sh
 alias curl="curl -s -k -u \"${NUODB_CP_USER}:${NUODB_CP_PASSWORD}\""
 ```
+
+### Setting up the Terraform Provider
+
+To use terraform to manage your databases, you will want to use the `nuodbaas` provider.

```

Comment:
Include a link to the NuoDBaaS Provider page in the [Terraform Registry](https://registry.terraform.io/providers/nuodb/nuodbaas).